preset/variables.scss

```diff
@@ -118,7 +118,7 @@ $headings: (
   )
 );
 
-$font-weights: (
+$fontWeights: (
```

Comment:
Ok I have reverted this now! I was thinking this when I went to convert the rest and then thought they were probably Vuetify specific... Do you know if there's a comment we can add to mute this lint error then? I had a quick look and I don't think it's like ruby where we can add comments to silence lints. Looks like we would need to add configurations to our editor if we wanted to do it. But lemme know if there's an easy way but otherwise happy to ignore it for now